gst_client/gst_client_symbolic_link.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# This script is executed at install time to create and install symbolic links
+
+# $1: target
+# $2: link name
+rm -f ${DESTDIR}${MESON_INSTALL_PREFIX}/$2
```

Comment:
Would it be saver to add a `/` between `DESTDIR` and `MESON_INSTALL_DIR`?

Reply:
The `/` is not required since `MESON_INSTALL_DIR` is an absolute path, then is mandatory to start with  `/`.
If `MESON_INSTALL_DIR` is not an absolute path, meson configuration process fails.